horovod/runner/elastic/discovery.py

```diff
@@ -15,20 +15,33 @@
 
 import io
 import logging
+import random
 import threading
-
+import time
 from collections import defaultdict
 
 from horovod.runner.common.util import safe_shell_exec
 from horovod.runner.elastic.worker import HostUpdateResult
-
-
+# Maximum time hosts are in cooldown period
+COOLDOWN_UPPER_LIMIT_SECONDS = 5 * 60
+# Minimum time hosts are in cooldown
+COOLDOWN_LOWER_LIMIT_SECONDS = 10
+# Unit of increasing cooldown for repeat failure
+COOLDOWN_RANGE_SECONDS = [COOLDOWN_LOWER_LIMIT_SECONDS, COOLDOWN_UPPER_LIMIT_SECONDS]
```

Comment:
Constant `COOLDOWN_RANGE_SECONDS` is not used anywhere. What is it good for?

Reply:
Are these default values? Then the name should indicate this.

Reply:
Removed